docs/mobile/usecases-change-default-map-click-behavior.mdx

```diff
@@ -0,0 +1,317 @@
+---
+title: Change Default Map Click Behavior
+description: Geocortex Mobile - Learn how to configure Geocortex Mobile to change the default map click behavior
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import UseCaseContainer from "../../src/components/UseCaseContainer";
+import UseCaseCard from "../../src/components/UseCaseCard";
+import PrereqsSnippet from "./snippets/prereqs.mdx";
+
+Geocortex Mobile comes with default behavior that runs on a map click. The default behavior will identify and display details for results in the nearby area. But what if you want to replace the functionality with your own behavior? This can be accomplished by leveraging [Geocortex Workflow](https://apps.geocortex.com/workflow/designer/) and editing the [app config](concepts-app-config.mdx).
+
+:::note
+It's actually possible to configure the Map's `onClick` event in [Geocortex Web Designer](https://apps.geocortex.com/mobileviewer/designer/), by setting a workflow as the action for the [Maps click event](https://docs.geocortex.com/webviewer/latest/admin-help/Default.htm#gwv/component-settings.htm%3FTocPath%3DConfiguration%7CComponent%2520Settings%7C_____0). However, the point of this use case is to demonstrate the concept of configuring actions using the app config.
+:::
+
+## What you're Building
+
+We will modify the default map on click behavior to buffer the click geometry by a user provided distance and create a graphic which visualizes this buffer.
+
+<img
+    src={useBaseUrl("img/mobile-custom-map-click-workflow-finished.png")}
+    alt="Map Click Workflow with Buffer Behavior"
+/>
+
+## Prerequisites
+
+<PrereqsSnippet />
+
+## Initial Set up
+
+Start the Geocortex Mobile SDK with the following layout and app json. You should see a simple application with one layer, Fire Hydrants, which has a related table Fire Hydrant Surveys.
+
+Try clicking the map. It should return results for the point you clicked in the results list.
+
+<Tabs
+  defaultValue="layout"
+  values={[
+    { label: "Layout", value: "layout" },
+    { label: "App Config", value: "config" },
+  ]}>
+<TabItem value="config">
+
+```json title="app/app.json"
+{
+    "$schema": "..\\..\\ViewerFramework\\app-config\\mobile\\mobile-app-config.schema.json",
```

Comment:
Looks like there's more of these `$schema` kicking around. Weren't we going to nuke?

Reply:
I didn't nuke them in mobile, as currently the assumption in the Mobile docs is that you are working within the quickstart. In that case, it's useful to have a schema reference, else it is a little bit of a tricky property to discover on your own. The property is super important right now in the early stages while we don't have API docs for all the components.